docs/tutorials/advanced/mass-inference.md

```diff
@@ -0,0 +1,194 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+Mass inference
+------------------
+
+The ASE calculator is not necessarily the most efficient way to run a lot of computations. It is better to do a "mass inference" using a command line utility. We illustrate how to do that here. 
+
+In this paper we computed about 10K different gold structures:
+
+Boes, J. R., Groenenboom, M. C., Keith, J. A., & Kitchin, J. R. (2016). Neural network and Reaxff comparison for Au properties. Int. J. Quantum Chem., 116(13), 979–987. http://dx.doi.org/10.1002/qua.25115
+
+You can retrieve the dataset below. In this notebook we learn how to do "mass inference" without an ASE calculator. You do this by creating a config.yml file, and running the `main.py` command line utility.
+
+```{code-cell} ipython3
+! [ ! -f data.db ]  && wget https://figshare.com/ndownloader/files/11948267 -O data.db 
+```
+
+```{code-cell} ipython3
+! ase db data.db
+```
+
+You have to choose a checkpoint to start with. The newer checkpoints may require too much memory for this environment. 
+
+```{code-cell} ipython3
+from ocpmodels.common.model_registry import MODEL_REGISTRY
```

Comment:
This can now be changed to
```python
from ocpmodels.common.model_registry import available_pretrained_models
print(available_pretrained_models)
```

A tiny bit cleaner by hiding the urls from users.

Reply:
Same for `from ocpmodels.models import model_name_to_local_file`